Documentation/Books/Manual/Upgrading/Manually/Cluster.md

```diff
@@ -81,20 +81,28 @@ The following API calls will activate and de-activate the Maintenance mode of th
 
 You might use _curl_ to send the API call.
 
-Activate Maintenance mode:
+#### Activate Maintenance mode:
 
+`curl -u username:password <coordinator>/_admin/cluster/maintenance -XPUT -d'"on"'`
+
+For Example:
 ```
 curl http://localhost:7002/_admin/cluster/maintenance -XPUT -d'"on"'
+
 {"error":false,"warning":"Cluster supervision deactivated. 
 It will be reactivated automatically in 60 minutes unless this call is repeated until then."}
 ```
 **Note:** In case the manual upgrade takes longer than 60 minutes, the API call has to be resend.
 
 
-Deactivate Maintenance mode:
+#### Deactivate Maintenance mode:
+
+`curl -u username:password <coordinator>/_admin/cluster/maintenance -XPUT -d'"off"'`
```

Comment:
Only as a side note: The API is a bit strange, because we use HTTP verbs for some routes, but not here (like DELETE /_admin/cluster/maintenance similar to the shutdown route).